#! /bin/sh /usr/share/dpatch/dpatch-run
## z03_fix-rename-info-files.dpatch by  <yoush@debian.org>
##
## DP: fix incomplete rename-info-files patch

@DPATCH@
diff -urNad gcc-4.2-doc-non-dfsg-4.2.3.nf1~/gcc/fortran/gfortran.texi gcc-4.2-doc-non-dfsg-4.2.3.nf1/gcc/fortran/gfortran.texi
--- gcc-4.2-doc-non-dfsg-4.2.3.nf1~/gcc/fortran/gfortran.texi	2008-02-09 16:52:04.754267756 +0300
+++ gcc-4.2-doc-non-dfsg-4.2.3.nf1/gcc/fortran/gfortran.texi	2008-02-09 16:53:57.174750434 +0300
@@ -102,7 +102,6 @@
 @ifinfo
 @dircategory Software development
 @direntry
-* gfortran: (gfortran).                  The GNU Fortran Compiler.
 * @value{fngfortran}: (@value{fngfortran}).                  The GNU Fortran Compiler.
 @end direntry
 This file documents the use and the internals of
diff -urNad gcc-4.2-doc-non-dfsg-4.2.3.nf1~/gcc/doc/install.texi gcc-4.2-doc-non-dfsg-4.2.3.nf1/gcc/doc/install.texi
--- gcc-4.2-doc-non-dfsg-4.2.3.nf1~/gcc/doc/install.texi	2008-02-09 16:52:05.182239145 +0300
+++ gcc-4.2-doc-non-dfsg-4.2.3.nf1/gcc/doc/install.texi	2008-02-09 16:53:10.581866370 +0300
@@ -96,7 +96,7 @@
 @end ifinfo
 @dircategory Software development
 @direntry
-* gccinstall: (gccinstall).    Installing the GNU Compiler Collection.
+* @value{fngccinstall}: (@value{fngccinstall}).    Installing the GNU Compiler Collection.
 @end direntry
 
 @c Part 3 Titlepage and Copyright
diff -urNad gcc-4.2-doc-non-dfsg-4.2.3.nf1~/gcc/doc/standards.texi gcc-4.2-doc-non-dfsg-4.2.3.nf1/gcc/doc/standards.texi
--- gcc-4.2-doc-non-dfsg-4.2.3.nf1~/gcc/doc/standards.texi	2008-02-09 16:52:04.754267756 +0300
+++ gcc-4.2-doc-non-dfsg-4.2.3.nf1/gcc/doc/standards.texi	2008-02-09 16:52:05.522216416 +0300
@@ -188,8 +188,8 @@
 GNAT Reference Manual}, for information on standard
 conformance and compatibility of the Ada compiler.
 
-@xref{Standards,,Standards, @value{gfortran}, The GNU Fortran Compiler}, for details
+@xref{Standards,,Standards, @value{fngfortran}, The GNU Fortran Compiler}, for details
 of standards supported by GNU Fortran.
 
-@xref{Compatibility,,Compatibility with the Java Platform, @value{gcj}, GNU gcj},
+@xref{Compatibility,,Compatibility with the Java Platform, @value{fngcj}, GNU gcj},
 for details of compatibility between @command{gcj} and the Java Platform.
